### DISCUSSION TITLE
Allow `quarto use` to download to the current directory

### DISCUSSION THREAD

**Discussion by malcolmbarrett** [quarto-dev/quarto-cli#8593] [Feature Requests]
### Description

I use `quarto use` to make use of my custom template, but I almost always start with an RStudio Project, not a Quarto command. Quarto then fights with me about the directory already existing, making me create a subdirectory. The default behavior would be better if, instead, you confirmed you want it in an existing directory.

**Comment by mcanouil** [comment]
Your directory needs to be empty otherwise Quarto can't know what to do with possible conflicting files.
For instance, you can have extension for projects, which creates `_quarto.yml`, how Quarto should know if it should just erase what's already there or not.
That's intended behaviour here.

Cc @dragonstyle 

Anyhow, an issue has been opened to track this:
- https://github.com/quarto-dev/quarto-cli/issues/8595

**Reply by malcolmbarrett** [reply]
I understand the issue and that it's intentional. I'm proposing a better solution. Me confirming I want it there is a nice simple one (then, it's a me problem, not a Quarto problem), but there are other ways. If such a file is particularly concerning, Quarto can of course check for `_quarto.yml` and ask what to do or for confirmation to do some default behavior.

I bump into this *every* time. This is non-ergonomic for RStudio users because the idiomatic pattern is to start with an RStudio Project.

**Reply by mcanouil** [reply]
I perfectly understood the rational.
For instance, an RSTudio project is never empty (Rproj file/folder) which means you can't use `quarto use` in such setup. That's not very good.
